### PR TITLE
no avatar

### PR DESCRIPTION
**Pull request by evoX0**

### CODE REVIEW

File: src/atoms/SvgIcon/index.js [hunk 1]
```diff
@@ -60,6 +61,8 @@ const SvgIcon = props => (
 					return <DoubleArrow {...props} />;
 				case 'star':
 					return <Star {...props} />;
+				case 'avatar':
```
Comment (sergeyvlakh) [inline src/atoms/SvgIcon/index.js]: Maybe smthing like `DefaultAvatar` or `PlaceholderAvatar` ? I'm not good at naming things
@misund requesting your help :}

Comment (misund) [reply src/atoms/SvgIcon/index.js]: I think I'd go with `silhouette` or `user`

Comment (evoX0) [reply src/atoms/SvgIcon/index.js]: sure, let it be DefaultAvatar

Comment (misund) [reply src/atoms/SvgIcon/index.js]: In general, I think I would gravitate towards naming icons for what they actually depict (like silhouette, gear, envelope) in the shiny atom, and then adding the metaphor (like user, settings, email) wherever we're using it.

Comment (misund) [reply src/atoms/SvgIcon/index.js]: So in wolverine-frontend, for example, we could have a component
`const UserIcon = props => <Icon name="silhouette" title="User" />`

**Pull request merged**